bedrock/products/templates/products/vpn/download.html

```diff
@@ -37,121 +39,219 @@
 
 {% block content %}
 <main>
-  <div class="mzp-l-content">
-    <header class="vpn-download-header">
-      <h1>{{ self.page_title_full() }}</h1>
-      {% if block_download %}
+  <section class="vpn-downloads">
+    <div class="mzp-l-content">
+      <header class="vpn-download-header">
+        {% if block_download %}
+        <h1>{{ self.page_title_full() }}</h1>
         <p>{{ ftl('vpn-download-not-available-in-country') }}</p>
-      {% else %}
-        <p>{{ ftl('vpn-download-sub-heading', devices=connect_devices) }}</p>
-      {% endif %}
-    </header>
 
-    {% if not block_download %}
-      <ul class="vpn-download-list">
-        {% call picto(
-          base_el='li',
-          title=ftl('vpn-download-for-windows'),
-          image=resp_img(
-            url='img/products/vpn/download/platform-windows-black.svg',
-            optional_attributes={
-              'class': 'mzp-c-picto-image',
-              'width': '100',
-              'loading': 'lazy'
-            }
-          ),
-          body=True,
-        ) %}
-          <p>{{ ftl('vpn-download-for-windows-requirements') }}</p>
-          <a class="mzp-c-button mzp-t-secondary" href="{{ windows_download_url }}" data-cta-type="button" data-cta-text="VPN Download (Windows)">
-            {{ ftl('vpn-download-button-copy') }}
-          </a>
-        {% endcall %}
+        {% else %}
 
-        {% call picto(
-          base_el='li',
-          title=ftl('vpn-download-for-mac'),
-          image=resp_img(
-            url='img/products/vpn/download/platform-mac-black.svg',
-            optional_attributes={
-              'class': 'mzp-c-picto-image',
-              'width': '100',
-              'loading': 'lazy'
-            }
-          ),
-          body=True,
+        {% call split(
```

Comment:
Suggestion (non-blocking): This split component is placed inside an existing `mzp-l-content` container, so at smaller viewports the spacing / alignment gets a little odd. Suggest moving this out of the container, so it can run full width of the page as indented?

Ref: https://protocol.mozilla.org/components/detail/split--default.html

"Note: This component is intended to be a full-width section of a page, with an outer container that spans the width of the viewport and generous spacing above and below. Split has an inner container to define its content width, so don’t place Split inside another mzp-l-content [container](https://protocol.mozilla.org/components/detail/content-container). The nested spacing will get weird."

Reply:
Oop, I forgot this case :) I also went ahead and removed the header split component from a `mzp-l-content` container, and instead moved that container to wrap around the download options section